lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -52,6 +53,9 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @param {@link (RedirectRequest:type)}
      */
     async loginRedirect(request?: RedirectRequest): Promise<void> {
+        if (this.isBrowserEnvironment) {
```

Comment:
If we want to prevent events from happening server side, I think it would be cleaner to put this check in the `broadcastEvent` method itself.

Reply:
Done.